core/lib/via_btc_client/src/inscriber/mod.rs

```diff
@@ -33,6 +33,7 @@ use crate::{
 mod fee;
 mod internal_type;
 mod script_builder;
+pub mod test_utils;
```

Comment:
let's put `#[cfg(test)]` attribute here, before this row

Reply:
I was not able to use the attribute `#[cfg(test)]`. The via_btc_sender can not see the code decorated with that attribute. I think for now it's fine to keep it as it, correct me if I'm wrong but this will not be included when build the release version, right?
Probably, later we can move the testing logic in a separate crate.